packages/venia-concept/src/components/Checkout/label.js

```diff
@@ -1,25 +1,27 @@
-import React, { Component } from 'react';
+import React from 'react';
 import { bool, node, shape, string } from 'prop-types';
 
-import classify from 'src/classify';
+import { mergeClasses } from 'src/classify';
 import defaultClasses from './label.css';
 
-class Label extends Component {
-    static propTypes = {
-        children: node,
-        classes: shape({
-            root: string
-        }),
-        plain: bool
+const Label = props => {
+    const { children, plain, ...restProps } = props;
+    const classes = mergeClasses(defaultClasses, props.classes);
+    const elementType = plain ? 'span' : 'label';
```

Comment:
Thanks for keeping this. There are no comments in the file, but it's important not to have nested `<label>` elements for accessibility.